packages/account/src/Components/verification-status-action-button/verification-status-action-button.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Button, ButtonLink, Text } from '@deriv/components';
+import style from './verifications-status-action-button.module.scss';
+
+type TButtonLink = {
+    onClick?: never;
+    to: string;
+};
+
+type TButton = {
+    onClick?: () => void;
+    to?: never;
+};
+
+type TVerificationStatusActionButton = { button_text: JSX.Element } & (TButtonLink | TButton);
+
+const VerificationStatusActionButton = ({ button_text, onClick, to }: TVerificationStatusActionButton) => {
+    return (
+        <div className={''}>
```

Comment:
why do we need this empty classname?

Reply:
Don’t need.. just accidentally left it here and didn’t removed while refactoring.
Thanks 👍